### PR TITLE
chore: add an option to launch cluster with bad RTC state

### PR DESCRIPTION
**Pull request by smira**
This is useful for time sync testing.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/3819)

### CODE REVIEW

**Review by Unix4ever — APPROVED** [review]

File: cmd/talosctl/cmd/mgmt/cluster/create.go [hunk 1]
```diff
@@ -809,5 +812,6 @@ func init() {
 	createCmd.Flags().StringVar(&configPatch, "config-patch", "", "patch generated machineconfigs (applied to all node types)")
 	createCmd.Flags().StringVar(&configPatchControlPlane, "config-patch-control-plane", "", "patch generated machineconfigs (applied to 'init' and 'controlplane' types)")
 	createCmd.Flags().StringVar(&configPatchJoin, "config-patch-join", "", "patch generated machineconfigs (applied to 'join' type)")
+	createCmd.Flags().BoolVar(&badRTC, "bad-rtc", false, "launch VM with bad RTC state (QEMU only)")
```
Comment (Unix4ever) [inline cmd/talosctl/cmd/mgmt/cluster/create.go:815]: would be nice to put such test flags under some separate sandbox like:

```
-test-flags bad-rtc...
```

Comment (smira) [reply cmd/talosctl/cmd/mgmt/cluster/create.go:815]: yeah, agree, I would hide 80% of the flags somewhere :)

like `--expert-mode` )

**Pull request merged**